sdk/appconfiguration/azure-appconfiguration/README.md

```diff
@@ -256,6 +256,23 @@ logging.basicConfig(level=logging.DEBUG)
 
 Http request and response details are printed to stdout with this logging config.
 
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions require
```

Comment:
Nit: won't it be required for every contribution?

Reply:
We use same words for most (if not all) packages. Maybe that is the reason?